src/___new___/data/compareInfo.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable max-len */
+
+export default {
+  aboutInfo: [
+    {
+      title: 'Calico Open Source',
+      link: '/calico/latest/about/',
+
+      description: [
+        'A free, self-managed networking and security solution for containers, virtual machines, and native host-based workloads.',
+        'Provides networking, network policy, and IP address management capabilities for cloud-native applications.',
+      ],
+      linkDescription: 'Go to docs',
+      image: 'img/calico-open-source.svg',
+    },
+    {
+      title: 'Calico Enterprise',
+      link: '/calico-enterprise/latest/about-calico-enterprise',
+
+      description: [
+        'A paid, self-managed security and observability solution for containers and Kubernetes.',
+        'Extends Calico Open Source’s networking and network security capabilities to offer more advanced security and observability capabilities for organizations running Kubernetes at scale.',
+      ],
+      linkDescription: 'Go to docs',
+      image: 'img/calico-enterprise.drawio.svg',
+    },
+    {
+      title: 'Calico Cloud',
+
+      link: 'calico-cloud',
+      description: [
+        'A pay-as-you-go, SaaS application that provides comprehensive container security across the entire container lifecycle (build, deploy, runtime).',
+        'A fully-managed version of Calico Enterprise that adds container image scanning and advanced threat detection capabilities.',
+      ],
+      linkDescription: 'Go to docs',
+      image: 'img/calico-cloud-features.svg',
+    },
+  ],
+  bestFit: [
+    {
+      title: 'Calico Open Source',
+      costSupport: ['Free, self-managed'],
+      target: 'Users',
+      image: 'img/calico-logo.webp',
+
+      bestFit: [' - Who want best-in-class networking and network policy capabilities.'],
+    },
+    {
+      title: 'Calico Enterprise',
+      costSupport: ['Paid, self-managed'],
+      image: '/img/calico-cloud-logo.webp',
+
+      target: 'Enterprise teams',
+
+      bestFit: [
+        ' - Who need full control to customize their networking security deployment to meet regulatory and',
+        ' compliance requirements for Kubernetes at scale.',
+        ' - Who want Tigera Customer Support best practices for day zero to production best practices',
+        ' custom training and workshops, and Solution Architects to customize solutions.',
+      ],
+    },
+    {
+      title: 'Calico Cloud',
+      costSupport: ['Pay-as-you-go, SaaS'],
+      image: '/img/calico-enterprise-logo.webp',
+
+      target: 'Small teams',
+      bestFit: [
+        ' - Who need to manage the full spectrum of compliance in a web-based console for novice users:',
+        '- Secure clusters, pods, and applications',
```

Comment:
These bullets (starting with - Secure clusters, pods, and applications) are items under the lead bullet so need to be indented.

Reply:
That can do in a pinch. Better would be to add a space and one hyphen? I tried adding spaces, but they never displayed in the build.

Reply:
fixed like so:
![image](https://user-images.githubusercontent.com/1526993/234582912-26a4a1e6-8709-4cf8-b92c-c00559134cf8.png)